argoverse/sensor_dataset_config.py

```diff
@@ -0,0 +1,41 @@
+from typing import NamedTuple, Optional
+
+from hydra.experimental import compose, initialize_config_module
+from hydra.utils import instantiate
+
+
+class ImageDimensions(NamedTuple):
```

Comment:
Can we make these `dataclasses` now that we support Python >3.6? I believe `omegaconf` uses `dataclasses` for their Structured Configs which `hydra` uses in the backend.